transport-service/src/main/java/org/microarchitecturovisco/transport/controllers/TransportsQueryController.java

```diff
@@ -201,4 +204,35 @@ public void consumeMessageDeleteTransportReservation(String requestJson) {
             transportCommandService.deleteReservation(command);
         }
     }
+
+    @RabbitListener(queues = "#{handleDataGeneratorCreateQueue}")
+    public void consumeDataGeneratorMessage(String requestJson) {
+        Logger logger = Logger.getLogger("TransportController");
+        logger.info("Got transport data generator: " + requestJson);
+
+        TransportUpdateRequest request = JsonReader.readDtoFromJson(requestJson, TransportUpdateRequest.class);
+
+        // create transport
+        if (request.getUpdateType() == DataUpdateType.CREATE) {
+            System.out.println("Created transport: " + request);
+
+            transportsQueryService.createTransport(request.getId(), request.getCourseId(), request.getDepartureDate(),
+                    request.getCapacity(), request.getPricePerAdult());
+            return;
+        }
+
+        Transport transport = null;
+        transport = transportsQueryService.getTransportById(request.getId());
```

Comment:
`        Transport transport = transportsQueryService.getTransportById(request.getId());`
        
        
        chyba lepiej tak to zapisac